haiku-apps/einsteinium/einsteinium-1.3.0.recipe

```diff
@@ -0,0 +1,97 @@
+SUMMARY="Monitor applications and system services"
+DESCRIPTION="Einsteinium provides smarter monitoring of applications and \
+system services for Haiku. It can restart applications and system services \
+that were quit or crashed, gather statistics on application usage and \
+provides customizable ranked lists of applications."
+HOMEPAGE="https://sourceforge.net/projects/esforhaiku/"
```

Comment:
Good catch, I've updated the URL and also removed the creation of the user_launch folder I no longer use.